helm/prometheus-rules/templates/kaas/tenet/alerting-rules/management-cluster.rules.yml

```diff
@@ -72,7 +72,7 @@ spec:
       annotations:
         description: '{{`Container {{ $labels.container }} in pod {{ $labels.namespace }}/{{ $labels.pod }} is restarting too often.`}}'
         runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/container-is-restarting-too-often/
-      expr: increase(kube_pod_container_status_restarts_total{cluster_type="management_cluster", container!="grafana.*|prometheus.*|promxy.*|promtail.*|silence.*|teleport.*|.*-admission-controller.*|*-aws-*|aws-*|route53-manager.*|cilium.*|external-dns.*|teleport.*|falco.*|kyverno.*|postgres.*|exception.*|edgedb.*|athena.*|app.*"}[1h]) > 5
```

Comment:
Why was teleport removed?

Reply:
it was already in the list, my last PR added it a second time as i missed it.

Reply:
I think it was removed because it was a duplicate